### PR TITLE
Persian sura name + Update Persian Translation + supportRtl

### PR DESCRIPTION
**Pull request by amirhakh**
salam

I made this change for better Persian & rtl support.
Please merge translation  & if recommended for you merge support rtl  layout.

و من الله توفیق
فی امان الله

### CODE REVIEW

**Comment by amirhakh** [comment]
salam
can add translation & audio from below links?
http://tanzil.net/trans/
http://zekr.org/resources.html

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,6 +2,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
 	package="com.quran.labs.androidquran"
 	android:installLocation="auto">
+	
+    <uses-sdk android:minSdkVersion="9"
```
Comment (ozbek) [inline app/src/main/AndroidManifest.xml]: uses-sdk is ignored, please do not add it here.

File: app/src/main/res/layout-ar-land/audio_panel.xml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layoutDirection="ltr"
```
Comment (ozbek) [inline app/src/main/res/layout-ar-land/audio_panel.xml]: `ltr`? Why? Also, please correct the indentation.

File: app/src/main/res/values-fa/readers.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+     <string-array name="quran_readers_name">
```
Comment (ozbek) [inline app/src/main/res/values-fa/readers.xml]: Please use 2 space for indentation, see Code style guide in readme for details.

File: app/src/main/res/values-fa/sura_names.xml [hunk 4]
```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="sura_names">
```
Comment (ozbek) [inline app/src/main/res/values-fa/sura_names.xml]: Same as in readers.xml, 2 space for indentation.

File: app/src/main/res/layout-ar-land/audio_panel.xml (hunk 2 above)
Comment (ahmedre) [inline app/src/main/res/layout-ar-land/audio_panel.xml]: I'll have to test this, but chances are, Amir is correct here. The problem is, layout mirroring enables when you enable the `supportsRtl` flag in the manifest - consequently, the behavior pre-api 17 and post-api 17 differs, especially when you use a phone in the Arabic locale. I ran into this in another app and ended up using this as a fix as well (despite it not being ideal, it was also the best way I could find).

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (amirhakh) [inline app/src/main/AndroidManifest.xml]: I was removed uses-sdk

File: app/src/main/res/layout-ar-land/audio_panel.xml (hunk 2 above)
Comment (amirhakh) [inline app/src/main/res/layout-ar-land/audio_panel.xml]: salam

supportRtl was made Rtl layout on 17+ and not act on prevuse android. This edit made Rtl actionbar, Notification & layout on 17+ that can disable on demand.
I add layoutDirection="ltr" for  arabic layout that customized for beter view. this wil disable Rtl layout on thouse layout and view as Ltr (disabled Rtl).
You can test this edite and view result for Arabic Or Persian layout (on 17+)?

در پناه حق
فی امان الله

**Comment by amirhakh** [comment]
salam

I was edit this file and correct indentation.

**Comment by amirhakh** [comment]
salam

Request Again !

can add translation & audio from below links?
http://tanzil.net/trans/
http://zekr.org/resources.html

**Comment by ozbek** [comment]
Wa alaikum assalam

@amirhakh 
Please post new requests to [Issues](https://github.com/quran/quran_android/issues) section.
(There are many similar open issues already, e.g., #323 might be of interest to you)

**Comment by amirhakh** [comment]
Salam
Ok

**Comment by ahmedre** [comment]
jazakAllah khair @amirhakh - btw, i added `android:layoutDirection="ltr"` to the `ViewPager` and `SlidingTabLayout` in `quran_index` until i can properly fix them in an upcoming patch insha'Allah.

**Comment by ahmedre** [comment]
manually merged.

**Comment by amirhakh** [comment]
سلام
کما لک «جزائک الله» ان شاء الله

You will merge new Farsi (Persian) translation & sura name?

**Comment by ahmedre** [comment]
if it was part of this request, i merged it (i just squashed all your commits into one). please see https://github.com/quran/quran_android/commit/cf38cba3d824f3307273622223d1d673905b064f

**Pull request closed without merging**